cli/filplus.go

```diff
@@ -427,7 +427,7 @@ var filplusListClaimsCmd = &cli.Command{
 			var claimList []map[string]interface{}
 
 			for key, val := range claims {
-				if ts.Height() > claim.TermMax+claim.TermStart || !cctx.IsSet("expired") {
+				if tsHeight > val.TermMax || !expired {
```

Comment:
@LexLuthr I think this should be `tsHeight > val.TermStart + val.TermMax`, @LexLuthr can confirm.

Reply:
@arajasek You are right. This should be val.TermStart + val.TermMax to filter out the expired claims.